.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 - MAX_LINE_LENGTH=100 bash ci/check_line_lengths.sh src/**/*.md
 install:
 - source ~/.cargo/env || true
-- cargo install mdbook --version '^0.4.3'
+- cargo install mdbook --version '^0.4.5'
```

Comment:
Doesn't `^0.4.3` mean 'minor version 0.4'? Is the point to enforce at least 0.4.5?

Reply:
Yea.  This was just to force a build to trigger.  Any other PR would have kicked it off, but just wanted to be explicit.